.github/workflows/build.yml

```diff
@@ -0,0 +1,107 @@
+# Terraform Provider testing workflow using different terraform versions
+# and clouds.
+
+name: Build
+
+# This GitHub action runs your tests for each pull request.
+on:
+  pull_request:
+    paths-ignore:
+      - "README.md"
+      - "Makefile"
+      - "project-docs/**"
+  push:
+    branches:
+      - "main"
+    paths-ignore:
+      - "README.md"
+      - "Makefile"
+      - "project-docs/**"
+
+
+# Testing only needs permissions to read the repository contents.
+permissions:
+  contents: read
+
+jobs:
+  # Ensure project builds before running testing matrix
+  build:
```

Comment:
please check if this is the same `build` in the `test_integration.yml` ?

Reply:
it's different, I could rename to install. Aren't jobs name local to the file?